serverlessworkflow/modules/ROOT/pages/release_notes.adoc

```diff
@@ -4,8 +4,18 @@
 == Notable changes
 
 // * link:https://issues.redhat.com/browse/KOGITO-XXXX[KOGITO-XXXX] - <description>
-
+* link:https://issues.redhat.com/browse/KOGITO-3921[KOGITO-3921] - Event state timeout
+* link:https://issues.redhat.com/browse/KOGITO-8068[KOGITO-8068] - Change default event marshaller/unmarshallers per channel
+* link:https://issues.redhat.com/browse/KOGITO-8159[KOGITO-8159] - Start event is not behaving as expected when exclusive is false
```

Comment:
I will move the start event one to bug fixes, since we only establish that this functionality is not supported in docs